CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to xc
+
+Thank you for using `xc` and for your interest in contributing. Please take a moment to get familiar with our guidelines.
+
+## Be Courteous
+We expect you to read and follow our [Code of Conduct](./CODE_OF_CONDUCT.md) and to conduct yourself always in a way that creates a positive environment and clearly demonstrates your courtesy.
+
+## Use GitHub Issues
+
+We track and discuss work using Issues. Please [open an issue](https://github.com/joerdav/xc/issues/new/choose) to discuss changes before sending a pull request. We may close a PR without review if it's opened without discussion, especially by an unknown contributor. Whether we choose to review a PR is based on a judgment call about the maintenance burden it adds.
+
+## Write Tests, Maintain Coverage
+
+We expect authors of net-new code to provide tests covering the new code. Prior to merging, CI will verify that all tests pass and that coverage equals or exceeds the pre-merge state.
+
+## Document Your Work
+
+We expect contributors to keep documentation (in `docs/`) up to date, reflecting all substantive changes. Use `xc run-docs` (requires `hugo`) to verify that your changes build successfully & look as you expect.
+
+# HOWTOs
+
+## Add Support For A Markup Language
+We aim to support popular, stable, lightweight plain-text markup languages. Our priorities for language support:
+- New languages should work as similar to the Markdown implementation as possible
+- Users should be able to read and maintain tasks without writing excessive markup or understanding implicit behavior
+- The implementer wants to actually maintain `xc` tasks using the new language (we won't add languages just to have them)
```

Comment:
It would be nice if people bringing a new language are willing to maintain their implementation. However, I don't expect for people to maintain their language that they implement initially. Which sort of means 2 things:

- People can contribute language implementations without any strings attached (woo).
- However! I'll only accept languages that I think are ubiquitous enough that I'm willing to take the maintenance burden.

A good litmus test is probably - if [github](https://github.com/github/markup/blob/master/README.md#markups) don't support the markup language, it is highly unlikely I'll accept the PR.

Reply:
I made a clarification here incorporating that link & info.